### DISCUSSION TITLE
dnscrypt mac X12 unable server certificates

### DISCUSSION THREAD

**Discussion by subberlin** [DNSCrypt/dnscrypt-proxy#2724] [Q&A]
hello,

my english is very bad. hours of searching answers for my problem.

3 machines: 2x mac X.8 (mountain lion) with dnscrypt 1.0.14 systemprefernces works fine.

1x mac X12 (montery) with dnscrypt 1.0.14 systemprefenrences do not work. maybe since adding, but i mentioned it today.

konsole: "kraftwerk dnscrypt-proxy[70932]: Unable to retrieve server certificate"

first question: systemprefences may not work in X12, so i need to use https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Installation-macOS

right?

answer from me: definitely not. systemprefences and nothing else, and definitely no homebrew

thanks for answer

**Comment by subberlin** [comment]
i remember that it worked because i needed to connect to router via ip and not name

**Comment by jedisct1** [comment]
`dnscrypt-proxy` 1.0.14 is very old and unmaintained. Version 2.x is completely different.

You should uninstall it, and indeed follow the instructions from https://github.com/DNSCrypt/dnscrypt-proxy/wiki

The current version is 2.1.5.

**Comment by subberlin** [comment]
thank you very much for your answer!!!

no, as i already told, i definitly do not do that like that. and it works at the moment, but i throw it away.

have a nice day!!!! bless you
